config/.env.example

```diff
@@ -1,12 +0,0 @@
-# Shared
```

Comment:
@Gold-Turtle ,

Please create this file under config/.env.example. In this pull request you have deleted this entire file, we need this file so other developers can refer to it to properly create their own config.